src/message/header/content_type.rs

```diff
@@ -1,4 +1,5 @@
 use std::{
+    convert::From,
```

Comment:
I don't think this line is necessary, as `From` is already part of the Rust prelude

Reply:
Oops, I'm not sure why that felt correct.